Makefile

```diff
@@ -1,6 +1,5 @@
 
 DOCKER ?= $(shell command -v docker)
-PANDOC ?= $(shell command -v pandoc)
```

Comment:
On Mon, May 23, 2016 at 12:23:18PM -0700, Vincent Batts wrote:
> Otherwise the whole command for the `docs` target will have to be
> overhauled to allow for a local pandoc binary.

How difficult would this be?  It looks like it's just moving
‘$(DOCKER) … -u $(shell id -u)’ into a PANDOC fallback (if the shell
can find a local ‘docker’ but not a local ‘pandoc’).